client/src/components/ConfigureWebhook/index.js

```diff
@@ -67,6 +73,57 @@ export default function ConfigureWebhook(props) {
     }
   };
 
+  const handleUpdateWebhook = async () => {
+    try {
+      setLoading(true);
+      const ghUpdateWebhookResponse = await axios({
+        method: "patch",
+        url: `${rc4gitApiDomain}/webhooks/github`,
```

Comment:
Since now we are using proxy, you must change it to relative URL, no need for domain name.
ANd also add /api